Makefile

```diff
@@ -10,7 +10,8 @@ INSTALL=install
 CFLAGS ?= -O2
 PICFLAG = -fPIC
 C99FLAG = -std=c99
-UCFLAGS = $(CFLAGS) $(PICFLAG) $(C99FLAG) -DUTF8PROC_EXPORTS
+WCFLAGS = -Wall -Wmissing-prototypes -pedantic
```

Comment:
Is `-Wmissing-prototypes` not included in `-Wall`? If so, why not? MSVC understands `-Wall` but not `-Wmissing-prototypes`, and currently my hacky method of building Julia with MSVC goes through makefiles rather than cmake for some deps.